xbmc/guilib/GUIControl.cpp

```diff
@@ -185,7 +186,15 @@ void CGUIControl::DoRender()
       g_graphicsContext.SetStereoFactor(m_stereo);
 
     GUIPROFILER_RENDER_BEGIN(this);
+
+    if (m_hitColor != 0xFFFFFFFF)
```

Comment:
I can't see the reason why m_hitColor is of tpye CGUIInfoColor. Why don't you make it uint32_t or colot_t?